homeassistant/components/frontend/www_static/polymer/components/camera-recording.html

```diff
@@ -0,0 +1,298 @@
+<link rel="import" href="../bower_components/polymer/polymer.html">
+<link rel="import" href="../bower_components/paper-button/paper-button.html">
+
+<polymer-element name="camera-recording" attributes="cameraImageEvent">
+<template>
+	<style>
+		:host .camera-event-thumb {
+			width: calc(90%);
+			display:inline;
+		}
+		:host .camera-thumb-list-item {
+			min-width: calc(10%);
+			max-width: calc(10%);
+			display:inline;
+		}
+
+
+		:host .playback-controls {
+			text-align:center;
+			text-align:-webkit-center;
+		}
+
+		:host .selected-frame {
+			border: solid #4285f4 2px;
+		}
+
+		:host .strip-container {
+			width:630px;
+		}
+
+        :host .camera-recording-container {
+            width:640px;
+            height:520px;
+        }
+
+        :host .camera-image-holder {
+            text-align: center;
+        }
+
+        :host .camera-event-image {
+            height:360px;
+
+        }
+
+		@media (max-width: 780px) {
+            :host .camera-event-image {
+                width: calc(100%);
+                height: initial;
+            }
+            :host .camera-recording-container {
+                width: calc(100%);
+                height: calc(100%);
+            }
+            :host .strip-container {
+                width: calc(100% - 10px);
+            }
+            :host .camera-thumb-list-item {
+                min-width: calc(20%);
+                max-width: calc(20%);
+                display:inline;
```

Comment:
Setting the width (or some other position-related property) of an inline element will cause the browser to then display the element as a block element. ([source](http://stackoverflow.com/a/1423338))